types/sse.d.ts

```diff
@@ -0,0 +1,90 @@
+export type SSE = {
+    /**
+     * - headers
+     */
+    headers: string;
+    /**
+     * - payload as a string
+     */
+    payload: string;
+    /**
+     * - HTTP Method
+     */
+    method: string;
+    /**
+     * - flag, if credentials needed
+     */
+    withCredentials: boolean;
+    /**
+     * - debugging flag
+     */
+    debug: boolean;
+    addEventListener: AddEventListener;
+    removeEventListener: RemoveEventListener;
+    dispatchEvent: DispatchEvent;
+    stream: Stream;
+    close: Close;
+    onmessage: OnMessage;
+    onopen: OnOpen;
+    onload: OnLoad;
+    onreadystatechange: OnReadystatechange;
+    onerror: OnError;
+    onabort: OnAbort;
+};
+export type SSEHeader = {
```

Comment:
I think correct name will be SSEHeaders
```suggestion
export type SSEHeaders = {
```